screens/help_screen.py

```diff
@@ -20,4 +24,26 @@ def keypress(self, size, key):
             self._emit('prev')
 
 
+title1 = "__      __           _    _        __ _        _____  _  _    _      \n" # noqa: CODE
+title2 = "\ \    / / ___  _ _ | |__(_) _ _  / _` |      |_   _|(_)| |_ | | ___ \n" # noqa: CODE
+title3 = " \ \/\/ / / _ \| '_|| / /| || ' \ \__. |        | |  | ||  _|| |/ -_)\n" # noqa: CODE
+title4 = "  \_/\_/  \___/|_|  |_\_\|_||_||_||___/         |_|  |_| \__||_|\___|\n" # noqa: CODE
```

Comment:
Could you just add the strings?
`title = "one"
          +"two"
          +"three"
          `